docs/howto/add_repo.rst

```diff
@@ -0,0 +1,38 @@
+Add a Package Repository
+************************
+
+It's possible to add your own apt repositories as sources for build-packages and
+stage-packages, including those hosted on a PPA, the Personal Package Archive,
+which serves personally hosted non-standard packages.
+
+Third-party apt repositories can be added to a Craft-based project by using
```

Comment:
> to a Craft-based project

Originally this said "to a snap's snapcraft.yaml file"; I guess it becomes a bit awkward to talk about project files without a specific application so if this reads "too weirdly" I'm definitely open to suggestions

Reply:
maybe "Craft Application" and maybe "(e.g.; Snapcraft, Rockcraft, Charmcraft)"

Reply:
I wanted to make a clearer distinction that this is about adding a repo to a file from a Craft Application, as opposed to adding "support" for repos to the Applications themselves, so I reworded it as:
> Third-party apt repositories can be added to the project file of a Craft Application (like Snapcraft, Rockcraft, or Charmcraft) by using the top-level package-repositories keyword ...

What do you think?

Reply:
Having a try at the whole sentence:

```
Craft applications can include third-party apt repositories using a top-level ``package-repositories`` keyword. These can be formatted as PPAs or as arbitrary deb repositories.
```

Reply:
remove "apt" and it is generic enough

Reply:
updated (removed apt)